pipelines/artefact-automation-pipelines/promote-between-environments-after-approval/pipeline.yaml

```
@@ -61,4 +61,4 @@ jobs:
       with:
         env_id: production
         set_id: ${{ steps.apply.outputs.set_id }}
-        comment: Deploying ${{ inputs.workloads[0] }}
\ No newline at end of file
+        comment: Deploying ${{ inputs.workloads[0] }}
```

I think this may need to be `inputs.workload_artefacts[0]`
```suggestion
        comment: Deploying ${{ inputs.workload_artefacts[0] }}
```